File: java/src/main/java/com/meteroid/models/FeatureStatus.java
// This file is @generated
package com.meteroid.models;

import com.fasterxml.jackson.annotation.JsonValue;
import com.meteroid.Utils.ToQueryParam;

public enum FeatureStatus implements ToQueryParam {
    ACTIVE("ACTIVE"),
    DISABLED("DISABLED"),
    ARCHIVED("ARCHIVED");
    private final String value;

    FeatureStatus(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return this.value;
    }

    @Override
    public String toQueryParam() {
        return this.value;
    }
}

File: python/meteroid/models/feature_status.py
# this file is @generated
import enum


class FeatureStatus(str, enum.Enum):
    """Lifecycle status of a feature."""

    ACTIVE = "ACTIVE"
    DISABLED = "DISABLED"
    ARCHIVED = "ARCHIVED"

    def __str__(self) -> str:
        return str(self.value)
